Explorer++/Explorer++/RegistrySettings.cpp

```diff
@@ -801,6 +801,14 @@ std::vector<Column_t> Explorerplusplus::LoadColumnFromRegistry(HKEY hColumnsKey,
 	for(i = 0;i < dwSize / sizeof(ColumnOld_t);i++)
 	{
 		column.type = static_cast<ColumnType>(columnList[i].id);
+
+		if (32 == static_cast<unsigned int>(column.type))

```

Comment:
Can you add a constant for 32 and use it here (as well as in XMLSettings.cpp).

Reply:
Potential ways to do that:

Add a new enum:
```
enum class DeprecatedColumnType 
{
    VirtualType = 32
}
```

or in `enum class ColumnType` add a new item `DeprecatedVirtualType = 32`

Any preference or other alternative?

Reply:
I'd be hesitant to add it to ColumnType, since it would make it easy to accidentally use the column, even though it's not available anymore. The first option sounds ok, though.

Reply:
PR updated.

Reply:
bump.